src/chart.js

```diff
@@ -169,9 +235,15 @@ var HPChart = CreateClass({
                             {selector[locale].supported_chartsortkeys}
                         </FormControl>
                         <span>CSV</span>
-                        <CSVLink className={"btn btn-default"} data={csvData}><span
+                        <CSVLink className={"btn btn-default"} data={csvData} filename={"motocal_charts.csv"}><span
+                            className={"glyphicon glyphicon-download-alt"} aria-hidden={"true"}></span></CSVLink>
+                        <button className={"btn btn-default"}><CopyToClipboard text={csvClipBoardData}><span
+                            className={"glyphicon glyphicon-copy"} aria-hidden={"true"}></span></CopyToClipboard>
+                        </button>
+                        <span>WIKI</span>
```

Comment:
"WIKI" should change to "NVD3" to make it more specific. (japanese wiki can't use this)

Reply:
Done